climate.py

```diff
@@ -258,3 +258,64 @@ def get_upload_status(upload_id, token, api_key):
         return res.json()
     else:
         return None
+
+def get_scouting_observations(token, api_key, x_limit=100, x_next_token=None, occurredAfter=None, occurredBefore=None):
+    """
+    Retrieve a list of scouting observations created or updated by the user 
+    identified by the Authorization header. 
+    https://dev.fieldview.com/technical-documentation/ for possible status
+    values and their meaning.
+    :param token: access_token
+    :param api_key: Provided by Climate
+    :param x-next-token: Opaque string which allows for fetching the next batch of results.
+    :param x-limit: Max number of results to return per batch. Must be between 1 and 100 inclusive.
+    :param occurredAfter: Optional start time by which to filter layer results.
+    :param occurredBefor: Optional end time by which to filter layer results.
+    :return: status json object containing scouting observation list and status.
+    """
+    uri = '{}/v4/layers/scoutingObservations'.format(api_uri)
+    headers = {
+        'authorization': bearer_token(token),
+        'accept': json_content_type,
+        'x-api-key': api_key,
+        'X-Limit': str(x_limit),
+        'X-Next-Token': x_next_token,
+        'occurredAfter': occurredAfter,
```

Comment:
`'occurredAfter'` and `'occurredBefore'` are query parameters. you can pass params in a requests.get call using the `params=...` key

Reply:
Accepted.